src/modal/modal-window.ts

```diff
@@ -1,20 +1,25 @@
 import {
+  AfterViewInit,
   Component,
-  Output,
+  ContentChild,
+  ContentChildren,
+  ElementRef,
   EventEmitter,
   Input,
-  ElementRef,
-  Renderer,
+  OnDestroy,
   OnInit,
-  AfterViewInit,
-  OnDestroy
+  Output,
+  Renderer
 } from '@angular/core';
 
 import {ModalDismissReasons} from './modal-dismiss-reasons';
 
+let nextId = 0;
+
 @Component({
   selector: 'ngb-modal-window',
   host: {
+    '[attr.aria-modal]': 'true',
```

Comment:
Hello Pawel,
   I know that this attribute is not added into Bootstrap modals, but I am taking this https://github.com/w3c/aria-practices/blob/master/examples/dialog-modal/dialog.html as reference. 
I found this after our phone call about the aria-selected, it was stored as bookmark not in my work google account. Just for information I will share same links here also.

Kind of specifications about accessible modals can be found here: https://cdn.rawgit.com/w3c/aria-practices/master/aria-practices.html#dialog_modal 
and details about this aria attribute here: http://w3c.github.io/aria/aria/aria.html#aria-modal

By the way I will open an issue in the Bootstrap repo to see if this has to be added or not. 

Thanks for the review.